ubuntu_from_source/Dockerfile

```diff
@@ -44,16 +36,33 @@ RUN ROOT_GIT_URL=https://github.com/root-project/root.git \
     ; fi \
     # Above lines will set ROOT_GIT_REVISION to ROOT_VERSION argument if it corresponds to a valid branch name (such as ROOT_VERSION=master),
     # otherwise it will assume its a semantic version string and try to convert it into the tag format (such as 6.26.00 -> v6-26-00)
-    && git clone --branch $ROOT_GIT_REVISION --depth=1 $ROOT_GIT_URL $ROOTSYS/source \
-    && cd $ROOTSYS/source \
-    && mkdir -p $ROOTSYS/build &&  cd $ROOTSYS/build \
-    && cmake $ROOTSYS/source \
-    -DCMAKE_INSTALL_PREFIX=$ROOTSYS \
+    && git clone --branch $ROOT_GIT_REVISION --depth=1 $ROOT_GIT_URL /tmp/source \

```

Comment:
If a tagged version is always used, a lighter option would be to use just wget to download, e.g. https://root.cern/download/root_v6.26.00.source.tar.gz.